### ISSUE TITLE
Issues with tibbles and `backCalc()`

### ISSUE THREAD

**Issue by droglenc** [droglenc/RFishBC#34]
`backCalc()` fails when the data is a `tibble` because the following lines return a one-column data.frame on `tibble`s but a vector on regular `data.frame`s. This can be corrected by converting the `tibble` to a `data.frame` with `as.data.frame()` but I need to find a more robust answer. Another possible solution, but I need to research this more, is to add `drop=TRUE` to the following lines in backCalc().

```
regLcap <- regdat[,rlang::quo_name(rlang::enquo(lencap))]
Lcap <- dat[,rlang::quo_name(rlang::enquo(lencap))]
```

**Comment by droglenc** [comment]
See this ...

```
library(tibble)

( dft <- tibble(id=1:10,x=rnorm(10)) )
( df <- as.data.frame(dft) )

dft[,"x"]
df[,"x"]
df[,"x",drop=FALSE]
dft[,"x",drop=FALSE]
```

**Comment by droglenc** [comment]
Use this is a test (seems to work properly for versions >0.2.0

```
data(SMBassWB2,package="RFishBC")
data(SMBassWB1,package="RFishBC")

smb1 <- inner_join(SMBassWB1,SMBassWB2,by="id") %>%
  arrange(id)
smb2 <- as_tibble(smb1)
str(smb1)
str(smb2)

smb1_FL <- backCalc(smb1,lencap,BCM="FRALE",inFormat="wide",digits=0)
smb2_FL <- backCalc(smb2,lencap,BCM="FRALE",inFormat="wide",digits=0)
str(smb1_FL)
str(smb2_FL)
```

**Comment by droglenc** [comment]
Is fixed in versions >0.2.0
